src/mbgl/map/map_context.hpp

```diff
@@ -40,13 +45,11 @@ class MapContext : public Style::Observer {
 
     void pause();
 
-    void resize(uint16_t width, uint16_t height, float ratio);
-
     using StillImageCallback = std::function<void(std::exception_ptr, std::unique_ptr<const StillImage>)>;
 
     void triggerUpdate(const TransformState&, Update = Update::Nothing);
-    void renderStill(const TransformState&, StillImageCallback callback);
-    RenderResult renderSync(const TransformState&);
+    void renderStill(const TransformState&, const FrameData&, StillImageCallback callback);
```

Comment:
What's the rationale for introducing a separate argument rather than making `framebufferSize` part of `TransformState`?